i18n/code.org/copy-untranslated-apps.rb

```diff
@@ -57,17 +59,21 @@
   'Romanian' => 'ro-RO',
   'Russian' => 'ru-RU',
   'Serbian (Cyrillic)' => 'sr-SP',
+  'Sinhala' => 'si-LK',
   'Slovak' => 'sk-SK',
   'Slovenian' => 'sl-SI',
   'Spanish' => 'es-ES',
+  'Spanish, Argentina' => 'es-AR',
   'Spanish, Mexico' => 'es-MX',
   'Swedish' => 'sv-SE',
   'Tamil' => 'ta-IN',
   'Thai' => 'th-TH',
   'Turkish' => 'tr-TR',
   'Ukrainian' => 'uk-UA',
   'Urdu (Pakistan)' => 'ur-PK',
-  'Vietnamese' => 'vi-VN'
+  'Uzbek' => 'uz-UZ',
+  'Vietnamese' => 'vi-VN',
+  'Zulu' => 'zu-ZA'
 }
 
 untranslated_apps = %w(
```

Comment:
does tutorialExplorer need to be added to this list as well?  Or was that a one-off local change?